packages/server/src/middleware/error.js

```diff
@@ -4,7 +4,7 @@ import consola from 'consola'
 
 import Youch from '@nuxtjs/youch'
 
-export default function errorMiddleware(err, req, res, next) {
+export default ({ resources, options }) => function errorMiddleware(err, req, res, next) {
```

Comment:
@pi0 is this really meant to be an arrow function that returns a function?

Reply:
@galvez Yes :) It is now a middleware factory that is made using a context. Previously we were using `bind(this)` for middleware which is not a good idea (Increases coupling)

Reply:
@pi0 coolio